hack/tools/release/notes/print.go

```diff
@@ -75,11 +78,23 @@ func (p *releaseNotesPrinter) print(entries []notesEntry, commitsInRelease int,
 		}
 	}
 
-	if p.releaseType != "" {
+	if p.releaseType != "" && !isPreReleasePrinted {
 		fmt.Printf("🚨 This is a %s. Use it only for testing purposes. If you find any bugs, file an [issue](https://github.com/%s/issues/new).\n", p.releaseType, p.repo)
+		isPreReleasePrinted = true
 	}
 
-	if p.printKubernetesSupport {
+	if p.releaseType != "" && previousReleaseRef.value == "" {
+		// This will add the release notes expansion functionality for a pre-release version
+		fmt.Print(`<details>
+<summary>More details about the release</summary>
+
+:warning: **RELEASE CANDIDATE NOTES** :warning:
```

Comment:
Should we change here between beta and rc release?

Reply:
It's for both, thanks for pointing out, made the changes.